main/src/test/java/eu/openg/guessnumberapi/main/GetGamesAcceptanceTest.java

```diff
@@ -0,0 +1,29 @@
+package eu.openg.guessnumberapi.main;
+
+import eu.openg.guessnumberapi.rest.route.RouteConstants;
+import kong.unirest.HttpResponse;
+import kong.unirest.Unirest;
+import org.junit.jupiter.api.Test;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+
+class GetGamesAcceptanceTest extends AcceptanceTestSetUp {
+
+    @Test
+    void getGamesFromDb() {
+        String url = "http://localhost:" + getPort() + RouteConstants.GAMES_PATH;
+        postNewGame(url);
+        HttpResponse<String> response = sendGetGamesRequest(url);
+        assertThatJson(response.getBody()).node("[0]").node("gameId").isEqualTo(10);
+        assertThatJson(response.getBody()).node("[0]").node("guessCount").isEqualTo(0);
+        assertThatJson(response.getBody()).node("[0]").node("guessCount").isNumber();
```

Comment:
tas is number kaip ir neturi prasmės, kai viena eilute aukščiau patikrini, kad jis nulis